src/parser/content-evaluator-module.ts

```diff
@@ -100,41 +100,21 @@ export class ContentEvaluatorModule implements Module {
   }
 
   _getOptimalModel(prompt: string) {
-    const encoder = encodingForModel("gpt-3.5-turbo");
+    const encoder = encodingForModel("gpt-4o");
     const totalSumOfTokens = encoder.encode(prompt).length;
 
     if (totalSumOfTokens <= 4097) {
       return "gpt-3.5-turbo";
     } else if (totalSumOfTokens <= 16385) {
-      return "gpt-3.5-turbo-16k";
+      return "gpt-3.5-turbo-16k"; // gpt-4o?
     } else {
       console.warn("Backup plan for development purposes only, but using gpt-4 due to huge context size");
-      return "gpt-4-turbo-preview";
+      return "gpt-4o";
     }
   }
 
   async _sampleRelevanceScoreResults(specification: string, comments: string[]) {
```

Comment:
Consider removing the function entirely to simplify the codebase and not leave artifacts.